linctl issue get crashes with nil pointer dereference

## Summary
`linctl issue get LOC-3812` crashes with a nil pointer dereference after printing issue details.

## Steps to reproduce
1. `linctl --version` → `0.1.0`
2. `linctl issue get LOC-3812`

## Expected
Command exits successfully after printing the issue.

## Actual
Crash with SIGSEGV / nil pointer dereference.

## Stack trace
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x10 pc=0x104c5788c]

goroutine 1 [running]:
github.com/dorkitude/linctl/cmd.init.func10(0x14000195400?, {0x1400003eb70, 0x1, 0x104ca8dac?})
	github.com/dorkitude/linctl/cmd/issue.go:625 +0x23bc
github.com/spf13/cobra.(*Command).execute(0x105180ae0, {0x1400003eb40, 0x1, 0x1})
	github.com/spf13/cobra@v1.8.0/command.go:987 +0x7d4
github.com/spf13/cobra.(*Command).ExecuteC(0x105181f00)
	github.com/spf13/cobra@v1.8.0/command.go:1115 +0x344
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/dorkitude/linctl/cmd.Execute()
	github.com/dorkitude/linctl/cmd/root.go:73 +0x24
main.main()
	github.com/dorkitude/linctl/main.go:14 +0x54
```

## Notes
- Crash reproduces consistently on macOS while fetching a Linear issue with attachments/comments.
- The command prints the issue details, then panics.

--- a/cmd/comment.go
+++ b/cmd/comment.go
@@ -83,7 +83,7 @@ var commentListCmd = &cobra.Command{
 				if i > 0 {
 					fmt.Println("---")
 				}
-				fmt.Printf("Author: %s\n", comment.User.Name)
+				fmt.Printf("Author: %s\n", commentAuthorName(&comment))
 				fmt.Printf("Date: %s\n", comment.CreatedAt.Format("2006-01-02 15:04:05"))
 				fmt.Printf("Comment:\n%s\n", comment.Body)
 			}
@@ -109,7 +109,7 @@ var commentListCmd = &cobra.Command{
 				// Header with author and time
 				timeAgo := formatTimeAgo(comment.CreatedAt)
 				fmt.Printf("%s %s %s\n",
-					color.New(color.FgCyan, color.Bold).Sprint(comment.User.Name),
+					color.New(color.FgCyan, color.Bold).Sprint(commentAuthorName(&comment)),
 					color.New(color.FgWhite, color.Faint).Sprint("•"),
 					color.New(color.FgWhite, color.Faint).Sprint(timeAgo))
 
@@ -160,7 +160,7 @@ var commentCreateCmd = &cobra.Command{
 			output.JSON(comment)
 		} else if plaintext {
 			fmt.Printf("Created comment on %s\n", issueID)
-			fmt.Printf("Author: %s\n", comment.User.Name)
+			fmt.Printf("Author: %s\n", commentAuthorName(comment))
 			fmt.Printf("Date: %s\n", comment.CreatedAt.Format("2006-01-02 15:04:05"))
 		} else {
 			fmt.Printf("%s Added comment to %s\n",

--- a/cmd/comment_helpers.go
+++ b/cmd/comment_helpers.go
@@ -1,0 +1,20 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/dorkitude/linctl/pkg/api"
+)
+
+func commentAuthorName(comment *api.Comment) string {
+	if comment == nil || comment.User == nil {
+		return "System"
+	}
+	if name := strings.TrimSpace(comment.User.Name); name != "" {
+		return name
+	}
+	if email := strings.TrimSpace(comment.User.Email); email != "" {
+		return email
+	}
+	return "System"
+}

--- a/cmd/issue.go
+++ b/cmd/issue.go
@@ -498,7 +498,7 @@ var issueGetCmd = &cobra.Command{
 			if issue.Comments != nil && len(issue.Comments.Nodes) > 0 {
 				fmt.Printf("\n## Recent Comments\n")
 				for _, comment := range issue.Comments.Nodes {
-					fmt.Printf("\n### %s - %s\n", comment.User.Name, comment.CreatedAt.Format("2006-01-02 15:04"))
+					fmt.Printf("\n### %s - %s\n", commentAuthorName(&comment), comment.CreatedAt.Format("2006-01-02 15:04"))
 					if comment.EditedAt != nil {
 						fmt.Printf("*(edited %s)*\n", comment.EditedAt.Format("2006-01-02 15:04"))
 					}
@@ -682,7 +682,7 @@ var issueGetCmd = &cobra.Command{
 			fmt.Printf("\n%s\n", color.New(color.FgYellow).Sprint("Recent Comments:"))
 			for _, comment := range issue.Comments.Nodes {
 				fmt.Printf("  💬 %s - %s\n",
-					color.New(color.FgCyan).Sprint(comment.User.Name),
+					color.New(color.FgCyan).Sprint(commentAuthorName(&comment)),
 					color.New(color.FgWhite, color.Faint).Sprint(comment.CreatedAt.Format("2006-01-02 15:04")))
 				// Show first line of comment
 				lines := strings.Split(comment.Body, "\n")

